data/presets/highway/track.json

```diff
@@ -30,29 +30,34 @@
     "terms": [
         "4wd",
         "4x4",
-        "agricultural road",
         "atv",
-        "carriage road",
         "double track",
-        "farm road",
-        "fire road",
-        "forest road",
         "four wheel drive",
         "high-clearance",
         "jeep",
-        "land access road",
-        "logging road",
         "offroad",
+        "off-road",
         "overgrown",
         "primitive",
         "quad",
-        "ranch road",
         "rut",
-        "track",
-        "track road",
-        "two track",
-        "unmaintained",
-        "woods road"
+        "unmaintained"
     ],
-    "name": "Track / Land-Access Road"
+    "name": "Track / Land-Access Road",
+    "aliases": [
+        "Agricultural Road",
+        "Carriage Road",
+        "Dirt Road",
+        "Land-Access Road",
+        "Farm Road",
```

Comment:
Glad we aren’t debating [the finer points](https://www.houstonpublicmedia.org/articles/news/2018/05/04/283588/whats-the-difference-between-ranch-to-market-and-farm-to-market-roads/) of American English anymore!

[![Farm Road 1](https://upload.wikimedia.org/wikipedia/commons/thumb/a/af/FM1-SignAlongRoadside.jpg/1024px-FM1-SignAlongRoadside.jpg)](https://commons.wikimedia.org/wiki/File:FM1-SignAlongRoadside.jpg)